background.js

```diff
@@ -19,12 +19,19 @@ function updateBranchName(tabId) {
   });
 }
 
+function updateBranchNameButton(tabId) {
+  chrome.tabs.sendMessage(tabId, {
+    addButton: true
+  });
+}
+
 chrome.tabs.onUpdated.addListener(function(tabId, change, tab) {
   if (change.status !== "complete") return;
 
   if (isGitHubIssueUrl(tab.url)) {
     chrome.pageAction.show(tabId);
     updateBranchName(tabId);
+    updateBranchNameButton(tabId);
```

Comment:
Feels weird that this is called `updateBranchNameButton`. The `updateBranchName` function is literally updating the branchName value. The `updateBranchNameButton` is essentially sending the message to append the button?

Reply:
Word, it is not obvious.  I'll change it to `signalAppendBranchNameButton` or something like that if we still need a separate message.

Reply:
Wound up merging the message methods and calling it `notifyContentOfUrlChange`  not sure if I like that name though.